portal/src/ThemePreviewWidget.module.css

```diff
@@ -126,6 +126,10 @@
   color: var(--color-text-unshaded);
 }
 
+.secondaryTxt {
+  color: var(--color-text-shaded-1);
```

Comment:
In dark theme, secondaryTxt is --color-text-shaded-6 according to authgear-dark-theme.css.